internal/node/hybrid/aws.go

```diff
@@ -15,9 +14,9 @@ import (
 	"github.com/aws/eks-hybrid/internal/ssm"
 )
 
-const iamRoleAnywhereProfileName = "hybrid"
-
 func (hnp *HybridNodeProvider) ConfigureAws(ctx context.Context) error {
+	configCtx, cancel := context.WithTimeout(ctx, 2*time.Minute)
+	defer cancel()
```

Comment:
that's going to start counting here instead of just before calling "wait for config"
if the ssm registration plus agent starts takes more than 2 mins, it wont wait for the creds appropiately